Formula/libisofs.rb

```diff
@@ -0,0 +1,35 @@
+class Libisofs < Formula
+  desc "Library to create an ISO-9660 filesystem with various extensions"
+  homepage "https://dev.lovelyhq.com/libburnia/libisofs"
+  url "http://files.libburnia-project.org/releases/libisofs-1.5.4.tar.gz"
+  sha256 "aaa0ed80a7501979316f505b0b017f29cba0ea5463b751143bad2c360215a88e"
+  license "GPL-2.0-or-later"
+
+  depends_on "libtool" => :build
+  depends_on "libzip"
+
+  def install
+    # Fix flat namespace usage.
+    inreplace "configure", "${wl}-flat_namespace ${wl}-undefined ${wl}suppress", "${wl}-undefined ${wl}dynamic_lookup"
+
+    system "./configure", *std_configure_args,
+                          "--prefix=#{prefix}"
```

Comment:
Already in the standard arguments
```suggestion
    system "./configure", *std_configure_args
```